main/local_modules/shared_web_theme/static/src/css/base.scss

```diff
@@ -14,4 +14,25 @@ body {
 
 html, body, #wrapwrap {
   height: 0 !important;
-}
\ No newline at end of file
+}
+//.grid-item { width: 25%; }
+
+.grid-item {
```

Comment:
Properties should be ordered background, color, float, height, margin, padding, position, width